mmv1/templates/terraform/examples/firestore_database.tf.erb

```diff
@@ -0,0 +1,37 @@
+resource "google_project" "project" {
+	provider = "google-beta"
```

Comment:
Nit: There is some inconsistent indentation in these example files, which will show up in the resource documentation website